run-ui.sh

```diff
@@ -10,11 +10,33 @@ if [[ -z "${OPENAI_API_KEY}" ]]; then
     exit 1
 fi
 
-cd /home/pn/py-proxy
-poetry run python py_proxy/proxy.py &
+# FIXME: If we unify Dockerfile and git dirs, this will become simpler.
+if [[ -f /.dockerenv ]]; then
+    PROXYDIR=/home/pn/py-proxy
+    UIDIR=/home/pn/js-ui
+else
+    cd $(dirname $0)
+    DIR=$(pwd)
+    PROXYDIR="${DIR}/openai-proxy"
+    UIDIR="${DIR}/ui"
+fi
+
+function cleanup {
+    kill "${PROXYPID}"
+}
+
+cd "${PROXYDIR}"
+: ${HOST:=localhost}
+openssl req -batch -x509 -newkey rsa:4096 -days 10000 \
```

Comment:
does this reuse keys if already created?

Reply:
No.  New ones every time.  In a container it doesn't matter.  For command-line use it would.  Are you thinking about avoiding extra friction in the browser?

Reply:
Just figured if we're rerunning it while developing we'd have a mess locally?